src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionFactsService.cs

```diff
@@ -48,4 +56,40 @@ public ImmutableArray<RazorCompletionItem> GetCompletionItems(RazorCompletionCon
 
         return completions.DrainToImmutable();
     }
+
+    // Internal for testing
```

Comment:
IMO just make this public. The class is already internal

Reply:
I'd prefer to leave this as is.